packages/forms-web-app/src/dynamic-forms/dynamic-components/multi-file-upload/question.test.js

```diff
@@ -22,5 +63,40 @@ describe('./src/dynamic-forms/dynamic-components/boolean/question.js', () => {
 		expect(multiFileQuestion.fieldName).toEqual(FIELDNAME);
 		expect(multiFileQuestion.validators).toEqual(VALIDATORS);
 		expect(multiFileQuestion.viewFolder).toEqual('multi-file-upload');
+		expect(multiFileQuestion.html).toEqual(HTML);
+	});
+	it('should render optional html correctly', () => {
+		const VIEWFOLDER = 'multi-file-upload';
+		const TEMPLATEPATH = `dynamic-components/${VIEWFOLDER}/index.njk`;
+		document.body.innerHTML = nunjucksTestRenderer.render(TEMPLATEPATH, viewModel);
```

Comment:
As mentioned above, if we have to support unnecessary code in the templates then I think the cost out weighs the benefit. They are also potentially brittle in that if we upgrade the gov uk packages for the nun jucks templates it could break these tests as the html elements could easily change.

Reply:
I have now removed the unit test